app/scripts/example-model.js

```diff
@@ -1,16 +0,0 @@
-'use strict';
-
-(function(exports) {
-  function ExampleModel(){
-    this.unicornList = ['Andrea','Merve','Jonathan','Van'];
-  };
-
-  ExampleModel.prototype = {
-    printHello: function(){
-      console.log('Hello');
-    }
-  };
-
-  exports.ExampleModel = ExampleModel;
-
-})(this);
```

Comment:
We feel maybe this should be retained, so that, as a user of the framework, as soon as I download it I can run it and play with it.

What do you guys think?